src/components/input.js

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { css, get, Input as input } from 'theme-ui';
+
+const variant = ({ theme, variant = 'primary', themeKey = 'placeHolderVariants' }) =>
+  css(get(theme, themeKey + '.' + variant, get(theme, variant)));
+
+const Input = styled(input)`
+  ::-webkit-input-placeholder { /* Chrome/Opera/Safari */
+    ${variant}
+  }
+  ::-moz-placeholder { /* Firefox 19+ */
+    ${variant}
+  }
+  :-ms-input-placeholder { /* IE 10+ */
+    ${variant}
+  }
+  :-moz-placeholder { /* Firefox 18- */
+    ${variant}
+  }
+`;
+
+export default Input;
```

Comment:
We need to implement placeholderVariants for other input types too. 
For example: Select, TextArea and etc